modules/porous_flow/test/tests/sinks/s12.i

```diff
@@ -0,0 +1,150 @@
+# Test PorousFlowEnthalpySink boundary condition with specified pressure
+
```

Comment:
Can you please add the following to the top of s12.i:
```
# The PorousFlowEnthalpy sink adds heat energy corresponding to injecting a 1kg/s/m^2 (flux_function = -1) of fluid at pressure 0.5 (given by the AuxVariable p_aux) and the input temperature is 300 (given by the T_in parameter).
# SimpleFluidProperties are used, with density0 = 10, bulk_modulus = 1, thermal_expansion = 0, and cv = 1E-4
# density = 10 * exp(0.5 / 1 + 0) = 16.4872
# internal energy = 1E-4 * 300 = 0.03
# enthalpy = 0.03 + 0.5/16.3872 = 0.0603265
# This is applied over an area of 100, so the total energy flux is 6.03265 J/s.
# This the the rate of change of the heat energy reported by the PorousFlowHeatEnergy Postprocessor
```

I hope this is right.  Please try it and tell me whether it works!

Reply:
The test reports `6.01231724399986` J/s with the changes now.

Reply:
Oh, i realised you should have
```
phase = 0
```
in your `PorousFlowHeatEnergy` Postprocessor.   That will give the correct 6.03265 J/s.

Reply:
Ok, adding `phase = 0` and I get `6.03265` J/s. Woot!

Reply:
✅

Reply:
@andrsd , i think you might have forgotten to re-gold with the 6.03265 J/s, hence the CIVET failures